_build_cfg.yml

```diff
@@ -445,6 +445,8 @@ Topics:
     Distros: openshift-origin,openshift-enterprise
   - Name: Limit Pod Access with Egress Firewall
     File: limit_pod_access_egress
+  - Name: Enabling Ingress Traffic by Exposing External Ports
```

Comment:
Consider rewording this description.  The feature enables assignment of a unique ip address to a service that, if routed correctly, enables external traffic to reach that service's ports.  'External Ports' doesn't make sense to me.